locales/de/subtitles/anita-malavasi/anita-malavasi-02.yml

```diff
@@ -261,17 +249,17 @@ de:
           s: 49
           from: 00:04:56,13
           to: 00:05:05,08
-          text: I had made friends with the young men who came to our bar.
+          text: Ich hatte Freunschaft geschlossen mit den jungen Männern, die in unsere Bar kamen.
         sequence-50:
           s: 50
           from: 00:05:05,08
           to: 00:05:10,29
-          text: When they left for war I became their communication point.
+          text: Als sie in den Krieg gingen, wurde ich ihr Kommunikationspunkt.
```

Comment:
Google Translate says `Bezugspunkt`.
What is the opinion of the Italian Experts(@busfaktor2 @ERA-aishah @eabalos @GContaldi @sgrippo @giulia1983)? Again the Italian transcript is available on http://www.resistance-archive.org/downloads/transcripts/anita-malavasi-it.pdf